pageserver/src/tenant.rs

```diff
@@ -442,6 +442,14 @@ impl WalRedoManager {
         }
     }
 
+    pub(crate) async fn ping(&self,pg_version: u32) -> Result<(), walredo::Error> {
```

Comment:
This would benefit from a comment that explains this is a test/observability thing and not part of our functional behavior